.circleci/config.yml

```diff
@@ -30,15 +30,13 @@ jobs:
                   conda create -n testenv --yes pip python=3.8
                   source activate testenv
                   conda install --yes scipy numpy matplotlib
-                  pip install mne pooch tqdm psutil mpi4py joblib nibabel
+                  pip install pooch tqdm mpi4py
```

Comment:
do you still need to install pooch and tqdm separately?

Reply:
We don't! Thanks for catching that!